### PR TITLE
Dataplane SLO Dashboard to use Recording rules

### PR DESCRIPTION
**Pull request by shivamG640**
## Description

Updated the SLO dashboard to use the recording rules. Also it has some minor changes to other dashboards

## Verification Steps

Let this PR get merged. Once it is merged, the changes can be verified on the stage grafana!

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
~~- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)~~
- [ ] Verified independently by reviewer
~~- [ ] Required Standard Operating Procedure (SOP) is added.~~

## Prometheus

~~- [ ] Changes were tested against dev environment~~

## Grafana

- [x] You have imported the changes into a dev or staging environment

### CODE REVIEW

File: resources/grafana/downstream/grafana-dashboard-dataplane-cluster-view.yaml [hunk 1]
```diff
@@ -2286,7 +2288,8 @@ data:
                 "mode": "absolute",
                 "steps": [
                   {
-                    "color": "green"
+                    "color": "green",
```
Comment (lgarciaaco) [inline resources/grafana/downstream/grafana-dashboard-dataplane-cluster-view.yaml:2291]: Can you revert all unnecessary changes like this one? It makes the PR harder to review.
```suggestion
                    "color": "green",
```

Comment (shivamG640) [reply resources/grafana/downstream/grafana-dashboard-dataplane-cluster-view.yaml]: @lgarciaaco Done!

File: resources/grafana/downstream/grafana-dashboard-dataplane-slos.yaml [hunk 2]
```diff
@@ -169,7 +172,7 @@ data:
                 "uid": "${Datasource}"
               },
               "editorMode": "code",
-              "expr": "( sum(rate(cos_fleetshard_sync_connector_state_count_total{cos_connector_state=\"ready\",cluster_id=~\"$Cluster\"}[28d])) by (cluster_id) /\n( sum(rate(cos_fleetshard_sync_connector_state_count_total{cos_connector_state=\"ready\",cluster_id=~\"$Cluster\"}[28d])) by (cluster_id) + sum(rate(cos_fleetshard_sync_connector_state_count_total{cos_connector_state=\"failed_but_ready\",cluster_id=~\"$Cluster\"}[28d])) by (cluster_id) ) ) * 100",
+              "expr": "slo_connector_availability_success_rate:ratio_rate28d",
```
Comment (lgarciaaco) [inline resources/grafana/downstream/grafana-dashboard-dataplane-slos.yaml:175]: Dont you have to still include the cluster_id?
```suggestion
              "expr": "slo_connector_availability_success_rate:ratio_rate28d{cluster_id=~\"$Cluster\"}",
```

**Review by lgarciaaco — APPROVED** [review]

File: resources/grafana/downstream/grafana-dashboard-dataplane-slos.yaml (hunk 2 above)
Comment (shivamG640) [reply resources/grafana/downstream/grafana-dashboard-dataplane-slos.yaml]: @lgarciaaco,
Added the "Cluster" variable back!
